spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -298,6 +298,7 @@
 "MPLv2.0, MIT Licences": MPL-2.0 AND MIT
 "MirOS License (MirOS)": MirOS
 "Mockrunner License, based on Apache Software License, version 1.1": Apache-1.1
+"Modified BSD License": BSD-3-Clause
```

Comment:
Modifications to the BSD license might require the introduction of a separate license ID.
Thus I have doubts this entry is valid without making assumptions about the package context. What do you think?

Reply:
@fviernau  BSD-3-Clause is actual known as the Modified BSD License see https://en.wikipedia.org/wiki/BSD_licenses#3-clause_license_(%22BSD_License_2.0%22,_%22Revised_BSD_License%22,_%22New_BSD_License%22,_or_%22Modified_BSD_License%22)

Reply:
Oh right, thanks!